### DISCUSSION TITLE
약관 테이블을 어떻게 구성할까 ?

### DISCUSSION THREAD

**Discussion by YurimYang** [SCBJ-7/SCBJ-BE#6] [Ideas]
### 안건

- 현재 서비스에서 요구하는 약관 사항이 "회원약관"과 "결제약관"이 있다.

- **"회원약관"** : 야놀자 계정과 연동을 할 것인지에 대한 약관
   🔗 참고 url : https://hyeyun133.tistory.com/entry/01-%ED%9A%8C%EC%9B%90%EA%B4%80%EB%A6%AC-%EB%8D%B0%EC%9D%B4%ED%84%B0
   
- **"결제약관"** : 
    1. 만 14세 이상 이용 동의
    2. 이용규칙 동의
    3. 취소 및 환불 규칙 동의
    4. 개인정보 수집 및 이용 동의
    5. 개인정보 제 3자 제공 동의 

### 고민사항
- 회원약관, 결제약관, 회원약관동의, 결제약관동의 총 4개의 테이블로 진행한다.
- 회원약관, 결제약관 테이블에 약관 동의관련된 칼럼을 집어넣는다.
- 회원, 결제 테이블에 약관 칼럼을 넣는다.

**Comment by Kwonminwoo** [comment]
약관이 추가되거나 변경될 가능성이 있으니 따로 테이블로 관리하는게 업데이트 할 때 편할 것 같네요!!!

**Reply by Aleexender** [reply]
따로 테이블 관리하는게 더 편할거 같다는 생각 저도 동의합니다!!

**Comment by wocjf0513** [comment]
해당 url에 있는 테이블이 잘 작성되있는 거 같아서, 그거랑 비슷하게 테이블 형성해도 될 거 같습니다!! 👍
